src/reedline_config.rs

```diff
@@ -123,6 +123,41 @@ pub(crate) fn add_history_menu(line_editor: Reedline, config: &Config) -> Reedli
     line_editor.with_menu(Box::new(history_menu))
 }
 
+fn add_menu_keybindings(keybindings: &mut Keybindings) {
+    keybindings.add_binding(
+        KeyModifiers::CONTROL,
+        KeyCode::Char('x'),
+        ReedlineEvent::UntilFound(vec![
+            ReedlineEvent::Menu("history_menu".to_string()),
+            ReedlineEvent::MenuPageNext,
+        ]),
+    );
+
+    keybindings.add_binding(
```

Comment:
according to fernando, this one (menupageprevious) for history doesn't need to be a untilfound event. i think it needs to be something like this:
```
    keybindings.add_binding(
        KeyModifiers::CONTROL,
        KeyCode::Char('z'),
        ReedlineEvent::MenuPagePrevious,
    );
``` 
tbh - i'm not sure it makes a big difference - don't know enough about how they're all hooked up.

Reply:
it does since undo is also bound to ctrl+z. Without it, ctrl+z will only do the menu previous